airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -128,3 +128,8 @@
   dockerRepository: airbyte/source-zendesk-singer
   dockerImageTag: 0.1.0
   documentationUrl: https://hub.docker.com/r/airbyte/source-zendesk-singer
+- sourceDefinitionId: d8313939-3782-41b0-be29-b3ca20d8dd3a
```

Comment:
I was mistaken about this before: for now we need to add _both_ the json and yaml entries 🤦 we'll fix this soon.